pkg/clients/helm/client.go

```diff
@@ -74,7 +74,7 @@ type client struct {
 }
 
 // NewClient returns a new Helm Client with provided config
-func NewClient(log logging.Logger, config *rest.Config, namespace string, wait bool, timeout time.Duration) (Client, error) {
+func NewClient(log logging.Logger, config *rest.Config, namespace string, wait bool, timeout time.Duration, skipCRDs bool) (Client, error) {
```

Comment:
Nit: 
With the number of parameters increasing in the function signature here, I believe we should switch using [variadic functional options](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis).

Reply:
@turkenh amended